common/static/common/js/spec/components/list_spec.js

```diff
@@ -0,0 +1,67 @@
+define(['jquery', 'backbone', 'underscore', 'common/js/components/views/list'],
+    function ($, Backbone, _, ListView) {
+        'use strict';
+        describe('ListView', function () {
+            var Model = Backbone.Model.extend({
```

Comment:
Nit: I prefer complex definitions to live outside the ``var`` declaration. It makes it hard to step through the code, and the indentation is confusing.